src/VCF.cpp

```diff
@@ -1,256 +1,257 @@
 // [[Rcpp::depends(RcppArmadillo)]]
-#include <RcppArmadillo.h>
```

Comment:
Are the changes in this file just formatting/C++ version stuff, or is there something  substantive that needed to be changed in the VCF backend?

Reply:
I must have committed this by accident. I reformatted a lot of the files while working on this to help me understand the code, but I did not mean to include them. I will fix this.